test/library/packages/EpochManager/sharedMemory/ReclaimedLockFreeQueue.chpl

```diff
@@ -0,0 +1,113 @@
+use EpochManager;
```

Comment:
This entire module needs to be moved to `module/package` with its test split into its own file.